spec/openapi.yaml

```diff
@@ -0,0 +1,271 @@
+openapi: 3.0.1
+info:
+  title: PrusaLink Web
+  description:
+    This is a printer webservice.
+  contact:
+    email: link@prusa3d.cz
+  license:
+    name: AGPLv3
+  version: 1.0.0-draft
+externalDocs:
+  description: Find out more about API
+  url: http://connect.prusa3d.cz
+security:
+  - digestAuth: []
+paths:
+  /api/version:
+    get:
+      summary: api version informations
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Version"
+
+  /api/v1/info:
+    get:
+      summary: printer informations
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Info"
+        401:
+          $ref: "#/components/responses/Unauthorized"
+
+components:
+  schemas:
+    Version:
+      type: object
+      properties:
+        api:
+          type: string
+          example: "1.0.0"
+        version:
+          type: string
+          example: "0.7.0"
+        printer:
+          type: string
+          example: "1.3.1"
+        text:
+          type: string
+          example: "PrusaLink 0.7.0"
+        firmware:
+          type: string
+          example: "3.10.1-4697"
+        sdk:
+          type: string
+          example: "0.7.0"
+      required:
+        - api
+        - version
+        - printer
+        - text
+        - firmware
+
+    Info:
+      type: object
+      properties:
+        printer_type:
+          type: string
+          example: "1.3.1"
+        firmware:
+          type: string
+          example: "3.10.0-4586"
+        mmu:
+          type: boolean
+          example: false
+        name:
+          type: string
+          example: "MuadDib"
+        location:
+          type: string
+          example: "Arrakis"
+        farm_mode:
+          type: boolean
+          example: false
+        serial:
+          type: string
+          example: "CZPX4720X004XC34242"
+        sd_ready:
+          type: boolean
+          example: true
+        hostname:
+          type: string
+          example: "prusa-mk3.lan"
```

Comment:
So, the info is one-off, same as version? What's the conceptual difference between version and info?